plugin/storage/integration/cassandra_test.go

```diff
@@ -77,6 +87,9 @@ func (s *CassandraStorageIntegration) initializeCassandra() error {
 	if s.SpanReader, err = f.CreateSpanReader(); err != nil {
 		return err
 	}
+	if s.CleanUp, err = s.cleanUp(f.PrimarySession); err != nil {
```

Comment:
you can assign s.CleanUp function at suite construction time and it will be called automatically before each test.

Reply:
I didn't get it properly can you please explain it in more details
Do you mean to assign CleanUp at `newCassandraStorageIntegration` function level ? I think as we required to pass the `f.CassendraSession()`  function to cleanup function we cannot setup at suite level i.e `newCassandraStorageIntegration` function.